storage-proofs-core/src/api_version.rs

```diff
@@ -13,40 +12,40 @@ pub enum ApiVersion {
 impl ApiVersion {
     pub fn as_semver(&self) -> semver::Version {
         match self {
-            ApiVersion::V1_0_0 => Version::new(1, 0, 0),
-            ApiVersion::V1_1_0 => Version::new(1, 1, 0),
+            ApiVersion::V1_0_0 => semver::Version::new(1, 0, 0),
```

Comment:
And here's a counter-example of my previously noted preference.  This change seems a bit odd to me, but having the added context is also ok.

Reply:
Please note that my comment is only about functions, it's not about structs. So my preference here would be to use `Version::new(1, 0, 0)1 and not `semver::Version::new(1, 0, 0)`.

Reply:
Yea, this was just a personal style choice. I'll revert it.

I made this change because while reading the code quickly, it was easy for me to mix up `ApiVersion` and `Version`, so adding the `semver::` prefix felt more explicit.